### PR TITLE
Updated current maintainers and our team page

### LINKED ISSUE #666: Include Mano & Ragha in the current maintainers' list
## Describe the desired feature/functionality
In the "CURRENT MAINTAINERS" list on the home page, include both @raghavenc5 and @Manoj1907

## Suggested implementation details

Add to https://github.com/etn-ccis/blui-doc-it/blob/dev/src/__configuration__/contributors/currentMaintainers.ts 

Optionally include photos too

### PR DESCRIPTION
**Pull request by yazadsidhwa-eaton**
Fixes #666 

#### Changes proposed in this Pull Request:

- Added in Ragha, Manoj, and Yazad to Current Maintainers on Landing Page
- Updated the Our Team page with all information

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: src/__configuration__/contributors/currentMaintainers.ts [hunk 1]
```diff
@@ -62,16 +63,24 @@ export const currentMaintainers: CurrentMaintainter[] = [
     {
         name: 'Raghavendran Chithirarajan',
         role: `Front-End Developer`,
+        image: RaghavC,
+        description:
+            'He loves to code both frontend & backend to create beautiful performant products with delightful user experiences. He has a passion for learning and exploring new things, especially in the fields of technology. He is a friendly and empathetic person also enjoys helping others and making new friends. In his free time, he enjoys listening to music and playing outdoor games.',
```
Comment (huayunh) [inline src/__configuration__/contributors/currentMaintainers.ts:68]: I recommend running this through a grammar checker

File: src/__configuration__/contributors/currentMaintainers.ts [hunk 2]
```diff
@@ -62,16 +63,24 @@ export const currentMaintainers: CurrentMaintainter[] = [
...
+        contacts: {
+            github: '',
+            linkedIn: 'https://github.com/RaghavendranC-Eaton',
```
Comment (huayunh) [inline src/__configuration__/contributors/currentMaintainers.ts:71]: why is there a GitHub link in the linkedIn field?

**Review by huayunh — CHANGES_REQUESTED** [review]

**Review by huayunh — APPROVED** [review]

File: src/__configuration__/contributors/currentMaintainers.ts [hunk 3]
```diff
@@ -57,4 +60,35 @@ export const currentMaintainers: CurrentMaintainter[] = [
...
+        contacts: {
+            github: 'https://github.com/RaghavendranC-Eaton',
+        },
+    },
+    {
+        name: 'Manoj Lokesh',
+        role: `Front-End Developer`,
+        image: ManojLokesh,
+        description:
+            'Software developer who primarily works on the front-end technologies. His hobbies are playing cricket, travelling, listening to music and watching movies.',
```
Comment (JeffGreiner-eaton) [inline src/__configuration__/contributors/currentMaintainers.ts:78]: one "L" in traveling for US but ok in UK

**Comment by JeffGreiner-eaton** [comment]
Also for some reason we lost our mobile view from your previous PR
Currently it looks like this in this PR...

image

**Comment by yazadsidhwa-eaton** [comment]
Mobile view should be fixed automatically when this branch is merged with Dev

**Pull request merged**